src/app/shared/co2-help-text/co2-help-text.component.html

```diff
@@ -0,0 +1,84 @@
+<div class="my-2"
+    *ngIf="currentField == 'totalEmissionOutputRate' || currentField == 'eGridSubregion' || currentField == 'zipcode'">
```

Comment:
- This condition needs the other two fields energySource and fuelType added.
- Move the condition to the ts file. Instead of explicitly using an if, try declaring currentField as a string literal + union type with the 5 values. I suspect if it's any value other than those, it may fall through as undefined and give the functionality we want.
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types